src/app/application.cpp

```diff
@@ -278,7 +279,7 @@ void Application::torrentFinished(BitTorrent::TorrentHandle *const torrent)
         program.replace("%T", torrent->currentTracker());
         program.replace("%I", torrent->hash());
 
-        QProcess::startDetached(program);
+        std::thread ([program]() { return system(program.toStdString().c_str()); }).detach();
```

Comment:
what if user put this in the textbox:
`antivirus"*" && mv /a/* /b && ...`
The antivirus scan takes a long time and then move some large files (which is slow)...

1. I don't want to parse what users inputted and append `&` to every command.
2. If user put the commands in a file and invoke it, there will be a shell process created to execute the commands, this way if user exit qbt the command sequence won't be interrupted (since the shell process won't be affected in anyway)